components/ActionSheet/README.md

```diff
@@ -37,8 +37,6 @@ the screen and displays actions a user can take.
   - [Typical use](#typical-use)
 - [MDCActionSheetController vs. UIAlertControllerStyleActionSheet](#mdcactionsheetcontroller-vs.-uialertcontrollerstyleactionsheet)
 - [Extensions](#extensions)
```

Comment:
I think you may need to add the theming doc to this table of contents and re-run the ActionSheet docs generator.

Reply:
Done